### PR TITLE
Change AKS credentials format to JSON

### PR DESCRIPTION
**Pull request by clebs**
**Description**

Changes proposed in this pull request:

- Changed credentials format to JSON.
- Added validation when loading credentials.

**Related issue(s)**
https://github.com/kyma-project/cli/issues/656

### CODE REVIEW

File: provision/examples/azure/README.md [hunk 1]
```diff
@@ -10,12 +10,14 @@ This example shows you how to use Hydroform to provision an Azure cluster.
 
 To provision an Azure cluster you need:
 
-1. The **Subscription ID**, **Tenant ID**, **Client ID** and **Client Secret** of your Azure subscription stored in a TOML file:
-    ```toml
-    SUBSCRIPTION_ID = "<my-subscription-id>"
-    TENANT_ID = "<my-tenant-id>"
-    CLIENT_ID = "<my-client-id>"
-    CLIENT_SECRET = "<my-client-secret>"
+1. The **Subscription ID**, **Tenant ID**, **Client ID** and **Client Secret** of your Azure subscription stored in a JSON file:
```
Comment (kazydek) [inline provision/examples/azure/README.md:13]:
```suggestion
1. The **Subscription ID**, **Tenant ID**, **Client ID**, and **Client Secret** of your Azure subscription stored in a JSON file:
```

Comment (kazydek) [reply provision/examples/azure/README.md:13]: serial comma? ;)

Comment (kazydek) [reply provision/examples/azure/README.md:13]: I would not bold them if unless you specifically quote the exact parameter names (and from what I can see, you don't)

**Review by suleymanakbas91 — APPROVED** [review]
We should document this also in the migration guide.

File: provision/examples/azure/README.md (hunk 1 above)
Comment (clebs) [reply provision/examples/azure/README.md]: Fixed thanks!

**Review by kazydek — APPROVED** [review]

**Pull request merged**